packages/bpk-component-carousel/src/BpkCarousel.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow strict */
+
+import PropTypes from 'prop-types';
+import React, { type Node } from 'react';
+import Carousel from 'nuka-carousel';
```

Comment:
We should reuse the [bpk-component-page-indicator](https://github.com/Skyscanner/backpack/tree/main/packages/bpk-component-page-indicator) component in this one, so we only need the swipeable function, but this library has a complete implementation, including the design of indicators. Even though we can get the same design through style coverage, but in order to maintain the consistency of the design, we always need extra efforts in subsequent maintenance. I think this is not acceptable

As we talked about before, I am still think that [react-swipeable](https://github.com/FormidableLabs/react-swipeable) is the best dependency library for this component